_docs/examples/provider/README.md

```diff
@@ -25,33 +32,45 @@ mkdir akash-demo && cd akash-demo
 
 ### Kubernetes Cluster
 
-* A Kubernetes cluster with `kubectl` connected to it.
-* A route-able IP address for ingress on the cluster.
-* A "star" domain pointing to the cluster route-able IP.
+* Version: `1.19+`
+* A Kubernetes cluster with your local `kubectl` with admin authentication and authorization.
+* A wildcard A Record resolving to a cluster route-able IP. eg: `*.akashian.io`
+
+#### Option 1: [Akash Kubespray](https://github.com/ovrclk/kubespray/) 
+
+Akash has created an example Kubespray configuration to provision a Kubernetes installation on linux machines. Included are best practice configuration for CNI `calico` plugin, default Network Policies + Namespaces, `nginx` Ingress Controller, `metrics-server`, `cert-manager`, and Seccomp syscall restrictions.
+
+
+#### Option 2: GCP Raw VM + k3s Example
+
+*Warning: needs to be updated*
 
 See [here](./kube-gce.md) for seting up these prerequisites
 using [GCE](https://cloud.google.com/compute).
 
+### Kubernetes Ingress
+
 In this tutorial, we will be using the domain `akashian.io`.
 
 ```sh
 PROVIDER_DOMAIN=akashian.io
 ```
 
-This `PROVIDER_DOMAIN` variable is used to configure the Kubernetes Ingress Controller domain, so for this example, a Deployment's requests will be routed via a random subdomain, eg: `kswtibraxfdhlflhrg6ahe.akashian.io`. A wildcard subdomain DNS entr is necessary to support this, eg `*.akashian.io`. This address can be customized in the [Configure provider services](#configure-provider-services) file's `ingress-domain` field, eg: `ingress-domain=ingress.akashian.io`.
+This `PROVIDER_DOMAIN` variable is used to configure the Kubernetes Ingress Controller domain, so for this example, a Deployment's requests will be routed via a random subdomain, eg: `kswtibraxfdhlflhrg6ahe.akashian.io`. A wildcard subdomain DNS entry is necessary to support this, eg `*.akashian.io`. This address can be customized in the [Configure provider services](#configure-provider-services) file's `ingress-domain` field, eg: `ingress-domain=ingress.akashian.io`.
 
 ### Akash Network
 
 We'll be connecting to the testnet.
 
 ```sh
-export AKASHCTL_NODE=tcp://rpc.akashtest.net:26657
-export AKASHCTL_CHAIN_ID=testnet-v4
+export AKASH_NODE=tcp://rpc-edgenet.akashdev.net:26657
+export AKASH_CHAIN_ID=edgenet
+export AKASH_KEYRING_BACKEND=test
```

Comment:
Are these not read from the environment later on?  I see the flags are used everywhere.